.travis.yml

```diff
@@ -4,6 +4,7 @@
 # schema_dev gets run.
 ---
 sudo: false
+dist: precise
```

Comment:
Changing this file manually isn't a good idea since it gets auto-generated -- changes need to be made to `schema-dev.yml`.   

Unfortunately, `schema_dev` doesn't currently have the ability to emit `dist: precise`... but fortunately it doesn't seem to be needed?  I just ran your specs for 5.0 and 5.1 in another branch without `dist: precise` and they worked fine.   What errors were you seeing without it?